### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

More fixes to existing tests and add new tests. With this one we cover all the urls (GET), we miss the POST actions. like actually creating pages, and other things (next PRs).

### CODE REVIEW

File: tests/Functional/Admin/BlockAdminTest.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ public static function provideCrudUrlsCases(): iterable
         yield 'List Block' => ['/admin/tests/app/sonatapageblock/list'];
         yield 'Edit Block' => ['/admin/tests/app/sonatapageblock/1/edit'];
         yield 'Remove Block' => ['/admin/tests/app/sonatapageblock/1/delete'];
-        yield 'Compose preview Block' => ['/admin/tests/app/sonatapageblock/2/compose_preview'];
+        yield 'Compose preview Block' => ['/admin/tests/app/sonatapageblock/2/compose-preview'];
```
Comment (jordisala1991) [inline tests/Functional/Admin/BlockAdminTest.php:51]: IMO underscores on urls do not make sense, better use -

File: src/Admin/BaseBlockAdmin.php [hunk 2]
```diff
@@ -119,11 +118,6 @@ protected function configurePersistentParameters(): array
         ];
     }
 
-    protected function configureRoutes(RouteCollectionInterface $collection): void
-    {
-        $collection->add('view', $this->getRouterIdParameter().'/view');
```
Comment (jordisala1991) [inline src/Admin/BaseBlockAdmin.php:124]: This route was added 8 years ago, without any comment, but it was never implemented.

Comment (jordisala1991) [reply src/Admin/BaseBlockAdmin.php:124]: actually 12 years ago, probably on one of the first commits, probably never worked.

File: src/Controller/BlockAdminController.php [hunk 3]
```diff
@@ -86,10 +86,6 @@ public function createAction(Request $request): Response
     {
         $this->admin->checkAccess('create');
 
-        if (!$this->admin->isChild()) {
```
Comment (jordisala1991) [inline src/Controller/BlockAdminController.php:89]: This check was modified by me on a previous PR. IMO we can remove it. Create action is executable on by child admin OR by shared block admin.

Comment (jordisala1991) [reply src/Controller/BlockAdminController.php:89]: But the remaining one (BlockAdmin) does not have the create route, I have remove it.

**Review by VincentLanglet — CHANGES_REQUESTED** [review]
If I understand correctly you're changing the route name/url.

This is a Major BC break which should be in the Upgrade note then.

**Comment by jordisala1991** [comment]
> If I understand correctly you're changing the route name/url.
> 
> This is a Major BC break which should be in the Upgrade note then.

Added UPGRADE note.

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**